src/client/Microsoft.Identity.Client/Instance/Authority.cs

```diff
@@ -132,18 +135,23 @@ public static Authority CreateAuthority(AuthorityInfo authorityInfo)
             switch (authorityInfo.AuthorityType)
             {
                 case AuthorityType.Adfs:
+                {
                     return new AdfsAuthority(authorityInfo);
-
+                }
```

Comment:
From a style perspective, we copied the .NET Core repo styles. We might have relaxed the rules on this particular issue (1 line statements not needing braces). It keeps the code concise.

Reply:
Rolled back